### PR TITLE
{vis}[GCCcore/9.3.0] Xvfb v1.20.9, nettle v3.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/81a797ba4ee7e06761b1fdf6101f323b for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/f589e90f13f0636f2ee1fa6aceeccd7e for a full test report.

File: easybuild/easyconfigs/n/nettle/nettle-3.6-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'nettle'
+version = '3.6'
+
+homepage = 'http://www.lysator.liu.se/~nisse/nettle/'
```
Comment (Micket) [inline easybuild/easyconfigs/n/nettle/nettle-3.6-GCCcore-9.3.0.eb:6]:
```suggestion
homepage = 'https://www.lysator.liu.se/~nisse/nettle/'
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/183dc44093e531434ed0ccc800b88bb4 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**